heron/uploaders/src/java/com/twitter/heron/uploader/s3/S3Context.java

```diff
@@ -18,10 +18,12 @@
 import com.twitter.heron.spi.common.Context;
 
 public class S3Context extends Context {
-  public static final String HERON_UPLOADER_S3_BUCKET = "heron.uploader.s3.bucket";
   public static final String HERON_UPLOADER_S3_ACCESS_KEY = "heron.uploader.s3.access_key";
```

Comment:
Purely nit picky organization thing but can you keep `HERON_UPLOADER_S3_ACCESS_KEY` and `HERON_UPLOADER_S3_SECRET_KEY` grouped together instead of putting keys between then?